pykrige/ok.py

```diff
@@ -251,15 +272,31 @@ def __init__(self, x, y, z, variogram_model='linear', variogram_parameters=None,
                 self.variogram_function = variogram_function
         else:
             self.variogram_function = self.variogram_dict[self.variogram_model]
+
         if self.verbose:
             print("Initializing variogram model...")
+
+        # The 'use_psill' kwarg lets the user specify whether variogram_parameters, if specified,
+        # are given in terms of the full sill or the partial sill. The full sill is easier to work
+        # with as a user, but the code works better with the partial sill. If the 'use_psill' is False
+        # (i.e., user is feeding in the full sill), the code converts to partial sill before initializing
+        # the variogram model...

```

Comment:
These (and other) comments are extremely informative, and we should probably put them in a doc, as it's less likely that users would read through the docs.